kernel/kernel-util/src/main/java/org/sakaiproject/util/Validator.java

```diff
@@ -477,6 +477,14 @@ public static boolean checkResourceId(String id)
 
 	} // checkResourceId
 
+    /**
+     * Check for a valid filename.
+     * 
+     * @return true if valid, false if not
+     */
+    public static boolean checkFilename(String filename) {
```

Comment:
If you're adding this method and just throwing an exception every time you get a false from it, why not just make this method throw the exception? Then your previous patterns would make more sense.

Reply:
Makes sense to me, thanks.